plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/action/GitAction.java

```diff
@@ -44,17 +42,26 @@ public GitAction(String text,
     }
 
     protected boolean isGitRepository() {
-        boolean isGitRepository = false;
 
-        if (getActiveProject() != null) {
-            ProjectConfigDto rootProjectDescriptor = getActiveProject().getRootProject();
-            List<String> listVcsProvider = rootProjectDescriptor.getAttributes().get("vcs.provider.name");
+        if(appContext.getActiveProject() == null)
```

Comment:
*if*-block should be formatted in accordance with codestyle: https://github.com/codenvy/codenvy-dev-resources/wiki/Formatting#non-empty-blocks